includes/class-sensei-teacher.php

```diff
@@ -811,6 +817,60 @@ public function filter_queries( $query ) {
 		}
 	}
 
+	/**
+	 * Filter the learners query to show
+	 * only users that belong to the current logged teacher.
```

Comment:
Yep, I agree this could be improved (updated in c9f1018e02fd654ec388aecc3de1e67f4495b187). I've decided to not use "enrolled" because it's not technically accurate in the current state.